### ISSUE TITLE
PTB: TypeError - Cannot read properties of underfined (reading 'Provider')

### ISSUE THREAD

**Issue by zulc22** [rauenzi/BDPluginLibrary#187]
```
TypeError: Cannot read properties of undefined (reading 'Provider')
    at DiscordProviders (betterdiscord://plugins/0PluginLibrary.plugin.js:4674:41)
    at Sa (https://ptb.discord.com/assets/c8c05de95d43ea97a1ce.js:1781:54)
    at gu (https://ptb.discord.com/assets/c8c05de95d43ea97a1ce.js:1835:146)
    at Ic (https://ptb.discord.com/assets/c8c05de95d43ea97a1ce.js:1828:89)
    at mc (https://ptb.discord.com/assets/c8c05de95d43ea97a1ce.js:1828:17)
    at hc (https://ptb.discord.com/assets/c8c05de95d43ea97a1ce.js:1827:867)
    at uc (https://ptb.discord.com/assets/c8c05de95d43ea97a1ce.js:1824:983)
    at Fi (https://ptb.discord.com/assets/c8c05de95d43ea97a1ce.js:1764:577)
    at lc (https://ptb.discord.com/assets/c8c05de95d43ea97a1ce.js:1825:498)
    at https://ptb.discord.com/assets/c8c05de95d43ea97a1ce.js:1844:522
    at el (https://ptb.discord.com/assets/c8c05de95d43ea97a1ce.js:1844:561)
    at t.render (https://ptb.discord.com/assets/c8c05de95d43ea97a1ce.js:1848:205)
    at Popouts.initialize (betterdiscord://plugins/0PluginLibrary.plugin.js:4584:18)
    at new PluginLibrary (betterdiscord://plugins/0PluginLibrary.plugin.js:6238:49)
    at A.initializeAddon (betterdiscord/renderer.js:4:33541)
    at A.loadAddon (betterdiscord/renderer.js:4:7808)
    at A.loadAddon (betterdiscord/renderer.js:4:32796)
    at A.loadAllAddons (betterdiscord/renderer.js:4:10099)
    at A.initialize (betterdiscord/renderer.js:4:4521)
    at A.initialize (betterdiscord/renderer.js:4:31995)
    at Object.startup (betterdiscord/renderer.js:4:357022)
```

**Comment by rauenzi** [comment]
This has been fixed in v2.0.15. Either update manually or wait for the update to come through the system.

**Comment by EEize** [comment]
> This has been fixed in v2.0.15. Either update manually or wait for the update to come through the system.

Still doesnt work for me tho

**Comment by dznem** [comment]
It not working
